.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: android
```

Comment:
Would you mind adding a comment that links to documentation of this config file?

Reply:
Done